src/locales/en/translation.json

```
@@ -50,5 +54,19 @@
   },
   "search": {
     "no_results": "No results found"
+  },
+  "modals": {
```

Requests and suggestions:
- Avoid using numbers in token naming, please opt for naming that is descriptive of the text intentions.
`citation_1_header` --> `included_in_report`

-`modal` does not give context for the text intention. What about a namespace of `exports` for these tokens? Only a few seem to involve the success and otherwise provide no context for a translator. What about names like:
```suggestion
  "exports": {
  "export_successful": "Export successful",
  "observation_level_one": "Observation level data ({{count}} survey)",
  "credit_request": "Please credit the data owners",
```

- Avoid redundancy. 'Exports' is already in the namespaces: 
`export_success.metadata_export` --> `exports.project_metadata`


I'll add these snippets to the guidelines as part of our living documentation